pypowsybl/network.py

```diff
@@ -4751,3 +4751,38 @@ def _get_c_dataframes_and_add_voltage_level_id(network: Network, dfs: _List[_Opt
         else:
             c_dfs.append(_create_c_dataframe(df, metadata[i]))
     return c_dfs
+
+def replace_tee_point_by_voltage_level_on_line(network: Network, line1z_id: str, linez2_Id: str, linezp_id: str,
+                                  voltage_level_id: str, bbs_or_bus_id: str, line1c_id: str, linec2_id: str,
+                                  line1c_name: str = None, linec2_name: str = None) -> None:
+    """
+    This method transform the action done in the create_line_on_line function into the action done in the connect_voltage_level_on_line class :
```

Comment:
We need to have a 1-line short description as the first line, otherwise the doc will render ugly things.

Reply:
Done